bindings/wasm/identity_wasm/examples/src/1_advanced/12_advanced_transactions.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020-2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { IdentityClient, IotaDocument } from "@iota/identity-wasm/node";
+import { IotaClient } from "@iota/iota-sdk/client";
+import { TransactionDataBuilder } from "@iota/iota-sdk/transactions";
+import { getFundedClient, getMemstorage, NETWORK_URL } from "../util";
+
+/** Demonstrate how to create a DID Document and publish it. */
+export async function advancedTransaction(): Promise<void> {
+    const storage = getMemstorage();
```

Comment:
We don't have overlapping keys, but could we use separate storages to show the physical separation of the two?